src/reconcile.rs

```diff
@@ -0,0 +1,81 @@
+use crate::client::Client;
+use crate::error::Error;
+use k8s_openapi::api::core::v1::Pod;
+use kube::api::{ListParams, Meta, ObjectMeta};
+use kube_runtime::controller::ReconcilerAction;
+use std::time::Duration;
+
+pub type ReconcileResult<E> = std::result::Result<ReconcileFunctionAction, E>;
+
+pub fn create_requeuing_reconciler_action(secs: u64) -> ReconcilerAction {
```

Comment:
As these fns are part of the public api and used in for example the ZK operator I think they should have a brief comment.